### DISCUSSION TITLE
Don’t fall into the technical trap, practicality comes first

### DISCUSSION THREAD

**Discussion by xmyuntule** [community/community#139383] [New to GitHub]
In the current open source community, I have seen many projects with very advanced technology, but these open source projects are not suitable for practical applications. For example, I am currently looking for an e-commerce open source project. I have searched for many project examples, and without exception, the interface is great and the functions are very rich.
But what I want to say is that the current e-commerce market environment is no longer pure e-commerce. Users must be interested and able to communicate and discuss with other users to promote transactions. I named this project: "Interest E-commerce".
I am looking for such an open source project. If you have a suitable or similar one, please recommend it to me. I am very willing to pay to try it.

**Comment by resort3779** [comment]
you should consider sponsoring the dev of your choice
